safrole/safrole.asn

```diff
@@ -0,0 +1,115 @@
+-- Syntax for Safrole protocol test vectors
+
+SafroleModule DEFINITIONS ::= BEGIN
+
+-- Define basic types
+u8 ::= INTEGER (0..255)
+u32 ::= INTEGER (0..4294967295)
+
+-- Define constants
+validators_count INTEGER ::= 6
+epoch_length INTEGER ::= 12
```

Comment:
How bad would it be to produce vectors in line with the final numbers? We should definitely produce some, even if we leave in the smaller models.

Reply:
Not that bad. I'll produce a batch (`full`) with the same checks but with the final numbers.
The ones with smaller numbers (`tiny`) will come handy when/if someone wants to manually tweak some stuff.